.github/workflows/build.yml

```diff
@@ -265,7 +265,7 @@ jobs:
           key: ${{ matrix.os }}-${{ matrix.EVENT_MATRIX }}-v4
 
       - name: Prepare vcpkg
-        uses: lukka/run-vcpkg@v7
+        uses: lukka/run-vcpkg@v11
```

Comment:
@diogoteles08 how to forbid dependabot to update some action? We don't want to update this one, see https://github.com/libevent/libevent/pull/1657

Reply:
Hi @azat , thanks for the contact! Well I confess I'm not experienced in this either, but I'm taking a look at the "Dependabot commands and options" section of this PR description and I think the solution would be to comment `@dependabot ignore lukka/run-vcpkg` on this PR.

Can you please try it out?

Reply:
Sorry, for the delay, and thank you for the message. But I have a question, is it persistent or only for this PR?

Reply:
And plus, I would prefer some solution that will be visible in the code, not some random comment in PR

Reply:
Something like this I guess should work - https://github.com/libevent/libevent/pull/1678

Reply:
IIUC, that commenting solution would be persistent, but seems like you found an even better solution, nice job! I didn't know about this possibility, liked it.
Let me know if this doesn't work as expected.